### DISCUSSION TITLE
v24.3

### DISCUSSION THREAD

**Discussion by alex-w** [Stellarium/stellarium#3904] [General]
**The installers are located in the "Assets" section at the end of these release notes!**

[![Github Release](https://img.shields.io/github/downloads/Stellarium/stellarium/v24.3/total.svg)](https://github.com/Stellarium/stellarium/releases/v24.3)

## Release notes
The Stellarium Team has released version 24.3.

The major changes of this version:
- Performance improvements for computation of Solar system bodies positions
- Improvements in plugins and AstroCalc tools
- Updates in sky cultures
- Updates in GUI

Behind the scenes, many more minor issues were fixed.

TELESCOPE USERS

We have identified some bugs in the TelescopeControl plugin on
the Qt6-based releases.

Telescope users, please try whether this solves your problems.
If not, we are looking for those of you with programming skills to help
us fixing remaining issues. Reward possible!

## What's Changed
Full list of changes:
- Added ability a batch drawing of asteroid and comet markers (GH: #3847)
- Added tiny circles as optional markers for minor bodies (GH: #3847)
- Added multithreading computations of positions for Solar system bodies (GH: #3847)
- Added tooltip for new option in AstroCalc/Ephemeris tool
- Added ability to use a wider range of dates for AstroCalc/Ephemeris tool (GH: #3880)
- Added a reddish tint over the landscape when sun is low (GH: #3858, #3855)
- Added abort telescope slew signals to LX200 and NexStar telescopes (GH: #2754)
- Added setting dynamic limits for wide range of dates in AstroCalc tools
- Added ability to use a wider range of dates for AstroCalc/RTS tool 
- Added links for multi-opposition unnumbered objects and one-opposition objects into bookmarks in Solar System Editor plugin
- Added info for total candidates for adding in Solar System Editor plugin
- Added ability to use a wider range of dates for AstroCalc/Phenomena tool (GH: #1327)
- Added ability to hide the constellation art button (GH: #3830)
- Added context support for exoplanet detection methods and exoplanetary classes (GH: #3798)
- Added comments for tooltips (GH: #3786)
- Fixed coordinates in '20 Fun Naked-Eye Double Stars' script (GH: #3835)
- Fixed typos in Maya skyculture
- Fixed linking with <execution> which may require libtbb
- Fixed messing in solar eclipse maps (GH: #3894)
- Fixed TimeZone for Xochicalco (GH: #3897)
- Fixed missing old-fashioned storing of new settings (GH: #3847)
- Fixed saving new option for AstroCalc/Ephemeris tool
- Fixed the circumstances discovery for few comets
- Fixed parsing MPC large numbers in Solar System Editor plugin (GH: #3882)
- Fixed Andromeda lines for Modern (S&T) sky culture (GH: #3865)
- Fixed Calendars plugin
- Fixed support common names for sky cultures (GH: #3845)
- Fixed typos in sky cultures (GH: #3854, #3838, #3842, #3845)
- Fixed working Search Tool/Lists tool when list of asterisms is empty (GH: #3853)
- Fixed duplicate Satellites filter window (GH: #3825)
- Fixed landscape label drawing
- Fixed “group”/“groups” mistake in Satellites plugin (GH: #3826)
- Fixed mistake in novae catalog (GH: #3831)
- Fixed a string for StelObject info (GH: #3813)
- Fixed missing feature for limiting magnitude for Solar system bodies (GH: #3817)
- Fixed re-display data when "boundless" flag is changed in AstroCalc/Ephemeris tool (GH: #3811)
- Fixed sorting rules for phase and elongation in the AstroCalc/Ephemeris tool (GH: #3810)
- Fixed tooltip in AstroCalc/Graphs tool (GH: #3809)
- Fixed Andromeda lines for Modern (IAU) sky culture
- Fixed messages in tooltips for AstroCalc tools (GH: #3801)
- Fixed selection of objects in AstroCalc/WUT tool (GH: #3799)
- Fixed translation issue for multiple detection methods for one exoplanet (GH: #3798)
- Fixed translation issue for exoplanetary classes of potentially habitable exoplanets (GH: #3798)
- Fixed displaying of morphological descriptions for globular star clusters (GH: #3782)
- Fixed bug in TUI plug-in (GH: #3784)
- Fixed coordinates of SN 1181A
- Fixed custom user dir option
- Changed core: small speedup for Preetham atmosphere
- Changed core: avoid StelBottomBar redraws when not needed
- Changed core: speed-up InfoPanel
- Changed core: avoid some costly computations
- Changed core: Allow display of colors for each planet and per minor body type (GH: #3878, #3849)
- Changed core: Allow to draw planet orbits together with orbit of selected object (GH: #3878, #3849)
- Changed core: speedup for nomenclature items
- Changed core: use QLatin1String for string literal comparison for Solar system objects
- Changed core: simplify gridline intersect callbacks (GH: #3792)
- Changed core: Use QLatin1String for string literal comparison in Planet and SolarSystem classes
- Changed core: Restored code of QtCompress (optional; bundled) (GH: #3802)
- Changed core: Allow disabling StelAudioMgr at runtime (GH: #3775)
- Changed GUI: Obvious improvement for Oculars GUI
- Changed Exoplanets plug-in: The list of detection methods of exoplanet has been revised
- Changed Satellites plugin: setting the new defaults for timerate limits for displaying satellites (x1000 or ~16 minutes per second) for smooth rendering the scene (GH: #3890)
- Changed Satellites plugin: speedups of computations
- Changed Telescope Control plugin: Disable ASCOM on Qt6 (GH: #3887)
- Changed Telescope Control plugin: Disable sync button if telescope does not support it
- Updated default list of locations
- Updated default catalog of satellites
- Updated default catalog of pulsars
- Updated default catalog of novae
- Updated default catalog of exoplanets
- Updated default list of minor bodies
- Updated photometry data for satellites
- Updated discovery circumstances for comets and minor planets
- Updated planetary nomenclature data
- Updated translations
- Updated lists of interesting stars
- Removed old comets
- Removed erroneous DSO names
- Removed links to dead wiki (GH: #3874)
- Removed link to MPC Numbered objects list from bookmarks in Solar System Editor plugin (a huge data)
- Removed unused code
- Removed unneeded tooltips (GH: #3797, #3796)

## Packages notes
package | note
------------|---------
stellarium-24.3.tar.gz | Stellarium 24.3 for UNIX/Linux (source code). <br />__MD5:__ 375367ffdfbbe3e4d4443ecebacf97ae <br />__SHA256:__ c3ffb56a049061c7754bafab176146a2c4474ecede108d650f3c7551e1eae50a
stellarium-24.3.tar.bz2 | Stellarium 24.3 for UNIX/Linux (source code). <br />__MD5:__ 1b601909f727bfdd7d72ea5a70fa3962 <br />__SHA256:__ bf8f24496fdecceefd75ec4ee60f57f7973905d94e3cd4c195d4d188dc25c8ce
stellarium-24.3.tar.xz | Stellarium 24.3 for UNIX/Linux (source code). <br />__MD5:__ a2f0a8dd0f2aeb49c05fb38aa63f929b <br />__SHA256:__ 03dad30f4031bdc2d200146e7c587249ea75dbdb88c8ae13632829aac5540b08
stellarium-24.3.zip | Stellarium 24.3 for UNIX/Linux (source code). <br />__MD5:__ ceb89c2742f2d051ae9fdcf882025296 <br />__SHA256:__ dfad69ba4d1a24e738a9ed2fc3a2f167410f1f92c044355a4a8a0b3a387fa20a
stellarium-24.3-qt5-win32.exe | Stellarium 24.3 for Windows (x86_32; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017). <br /> __MD5__: d45dea6a3878cdb22676189b9bed8a11 <br />__SHA256__: 35ef9fe4c4217c5dd36d611b29382c5af61423edec9c7ef13c2b27ce6b80a5ad
stellarium-24.3-qt5-win64.exe | Stellarium 24.3 for Windows (x86_64; Windows 7+) based on Qt 5.12.6 (Visual Studio 2017). <br /> __MD5__: 3be9f2296cc0165faef74253b64bbb0f <br />__SHA256__: 60faba4a4570a7591e333836cec247bb1992719a04cb41c902194ba6f27394cb
stellarium-24.3-qt6-win64.exe | Stellarium 24.3 for Windows (x86_64; Windows 10+) based on Qt 6.5.3 (Visual Studio 2019). <br /> __MD5__: ffc5d67216eef800bc865ab40502c1b7 <br />__SHA256__: d137912a94c1f2c3f25b0b36f20e2e7d3610b337b854bcd82bb66dba61e38a01
stellarium-24.3-qt6-arm64.exe | Stellarium 24.3 for Windows (ARM64; Windows 10+) based on Qt 6.5.3 (Visual Studio 2022). <br /> __MD5__: 39a8856bf539e1b6efb33b82b2a554f5 <br />__SHA256__: 3615dbfca0e962b4ecc66f0010a2173337d87074f5ad4c05ec24b94547069512
Stellarium-24.3-qt6-macOS.zip | Stellarium 24.3 for macOS (universal; macOS 11.0+) based on Qt 6.5.3 (AppleClang 16.0.0). <br /> __MD5__: 99a660bba53af10703eef112c68d11b2 <br />__SHA256__: 52462403d3d5c16613c364b023efbddbb1b036140d60e68f70f67a6c0dc8e519
Stellarium-24.3-qt5-x86_64.zip | Stellarium 24.3 for macOS (x86_64; macOS 10.14+) based on Qt 5.12.12 (AppleClang 13.0.0). <br /> __MD5__: 24d42ee209cdab22cbc4f2b0722a98a2 <br />__SHA256__: 6e4fb74715010f213fa4bf55affa1661b572b4b9e1705aa3f882744293350394
Stellarium-24.3-qt5-x86_64.AppImage | Stellarium 24.3 for Linux (amd64; glibc 2.31) based on Qt 5.12.8 (GCC 9.4.0). <br /> __MD5__: 075ca71b9f07fef80f3d2984bcc52dbc <br />__SHA256__: 2c136618ede7fb0687d8371a95a004e4bf9f00e0a1a83359de88b04aeec1cf0b
Stellarium-24.3-qt6-x86_64.AppImage | Stellarium 24.3 for Linux (amd64; glibc 2.35) based on Qt 6.2.4 (GCC 11.4.0). <br /> __MD5__: be7710c2323197b0bbb637c3c6f6e92a <br />__SHA256__: 304764814caa5a23c0b848abc7e429e0ec1ef9e1f90ae999d9ef197c422562b3

### GPG signature
1. Import Stellarium's GPG key
  `curl -O https://stellarium.org/files/stellarium.gpg`
2. List Stellarium's GPG key
  `gpg --with-fingerprint --show-keys --keyid-format long stellarium.gpg`
  **id:** `rsa4096/BF38D4D02A328DFF 2012-04-07`
  **fingerprint:** `7915 1C2E 6351 E727 8DA1  A730 BF38 D4D0 2A32 8DFF`
3. Verify the binary file is valid (tarball for example)
  `gpgv --keyring ./stellarium.gpg stellarium-24.3.tar.gz.asc stellarium-24.3.tar.gz`

<hr /><em>This discussion was created from the release <a href='https://github.com/Stellarium/stellarium/releases/tag/v24.3'>v24.3</a>.</em>

**Comment by eseavey7** [comment]
I installed 24.3 and now the ASCOM radio button under Telescope Control Connection has been greyed-out.  Has the ASCOM functionality been removed completely?  I went back to 24.2 so that I can use ASCOM.

Thanks

**Reply by alex-w** [reply]
> I installed 24.3 and now the ASCOM radio button under Telescope Control Connection has been greyed-out. Has the ASCOM functionality been removed completely? I went back to 24.2 so that I can use ASCOM.

Please use Qt5-based builds

**Reply by gzotti** [reply]
@eseavey7 did you use ASCOM with a Qt6-based build before? Did it work without crashes? We only know what others tell us. Qt6 and ASCOM seem to not like each other.

**Reply by eseavey7** [reply]
Qt6 24.2 was the first Qt6 version I used that would unexpectedly crash.  Last month I was able to use it and imaged a few nights, but with the program randomly crashing on my Windows 10 machine.  Everything would have been great id it weren't for the random crashes.
I can install the Qt5 and let you know how that works.

**Comment by eseavey7** [comment]
I installed Qt5 24.3 and so far that is working on my Windows 10 machine.

Thanks.

**Comment by Dequinzr** [comment]
J'ai installé et testé la version 24.3 avec QT5 en Windows 11 et ça fonctionne sans problème
La version 24.3 avec QT6 grisait le choix Ascom comme Every7 rendant la gestion du télescope inutilisable.
